### PR TITLE
Set correct html lang

### PR DESCRIPTION
**Pull request by gustaveen**
## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2317** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2317" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2317?utm_source=stack-comment).

File: apps/store/src/utils/l10n/locales.ts [hunk 1]
```diff
@@ -8,6 +8,7 @@ export type LocaleData = {
   locale: IsoLocale
   language: Language
   routingLocale: RoutingLocale
+  htmlLang: 'sv-SE' | 'en-SE' | 'no-NO' | 'en-NO' | 'da-DK' | 'en-DK'
```
Comment (alebedev) [inline apps/store/src/utils/l10n/locales.ts:11]: Let's use `locale` above, it's the same value

**Review by alebedev — CHANGES_REQUESTED** [review]

File: apps/store/src/utils/l10n/locales.ts (hunk 1 above)
Comment (gustaveen) [reply apps/store/src/utils/l10n/locales.ts:11]: I kind of like being explicit on those values and in this case it differs on norwegian htmlLang `no-NO`

**Review by alebedev — APPROVED** [review]
Makes sense, let's keep separate field with a comment why we're doing it

**Pull request merged**